src/multiboot-headers.md

```diff
@@ -282,7 +282,21 @@ Again, we’ll use math to let the computer calculate the sum for us. We add up
 the magic number, the mode code, and the header length, and then subtract it
 from a big number. `dd` then puts that value into this spot in our file.
 
-TODO: https://github.com/intermezzOS/book/issues/28
+> By the way...
```

Comment:
Yeah, check this out, it's definitely [checked in as bold](https://raw.githubusercontent.com/dariusk/book/master/src/multiboot-headers.md). I'm going to assume it's a rendering issue with the diff here?